pkg/kubelet/cm/cpumanager/policy_static_test.go

```diff
@@ -91,6 +92,18 @@ func TestStaticPolicyStart(t *testing.T) {
 			stDefaultCPUSet: cpuset.New(1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11),
 			expCSet:         cpuset.New(1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11),
 		},
+		{
+			description:      "non-corrupted state (StrictCPUReservationOption)",
+			topo:             topoDualSocketHT,
+			cpuPolicyOptions: map[string]string{StrictCPUReservationOption: "true"},
+			stAssignments: state.ContainerCPUAssignments{
+				"fakePod": map[string]cpuset.CPUSet{
+					"0": cpuset.New(0),
+				},
+			},
+			stDefaultCPUSet: cpuset.New(1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11),
+			expCSet:         cpuset.New(1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11),
+		},
```

Comment:
in this test it seems we don't have a strictly reserved cpu set (or a reserved set at all). Let's add a comment about to clarify what we are testing here. Perhaps non-regression with indeed no reserved set at all? but do we allow this configuration?

Reply:
I think you're right; this test case doesn't make sense. This configuration is not allowed from what I understand now ("The kubelet requires a CPU reservation greater than zero (...) when the static policy is enabled" according to the docs).